app/src/main/java/de/qspool/clementineremote/backend/player/MyLibrary.java

```diff
@@ -205,6 +210,11 @@ public void getAllTitlesFromArtistAsync(String artist) {
         this.execute(LVL_TITLE);
```

Comment:
Yes, because once `RELEASE` ordering was selected, `getAllTitlesFromArtistAsync(String) will also sort by `RELEASE`. When I'd call this method without a specific ordering, i'd expect the default odering instead of the last one used.

Same reason for the other methods.